core/sail/elasticsearch-store/src/main/resources/META-INF/services/org.eclipse.rdf4j.sail.config.SailFactory

```diff
@@ -0,0 +1 @@
+org.eclipse.rdf4j.sail.extensiblestore.config.ElasticsearchSailFactory
```

Comment:
I think this needs to be `org.eclipse.rdf4j.sail.elasticsearchstore.ElasticsearchSailFactory`.

Reply:
Good catch thanks :)